automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -638,7 +619,25 @@ groups:
       summary: "One or more {{ $labels.testnet }} nodes are stuck at an old block height (Observed block height did not increase in the last 30m)"
       description: "{{ $value }} blocks have been validated on network {{ $labels.testnet }} in the last hour (according to some node)."
       runbook: "https://www.notion.so/minaprotocol/FewBlocksPerHour-47a6356f093242d988b0d9527ce23478"
- 
+
+  - alert: StuckInBootstrap
+    expr: max by (testnet) (increase(Coda_Runtime_process_uptime_ms_total{${berkeley_testnet},syncStatus = "BOOTSTRAP"}[2h])) >= 6000000
```

Comment:
Just a note: I think this approach may trigger this alert if a node that was in bootstrap exits bootstrap and then re-enters bootstrap again, since the `rate` function in prometheus do not skip breaks in data, they only filter out drops in values. When you apply the `syncStatus` filter here, you are causing a break in the data, but when the data matches the filter again, there will be a massive observed jump in the `process_update_ms_total` between the breaks, which will still be observed in the final output of this query.

Let's merge this for now, as we need a new metric to fix this. Something like `current_sync_status_process_uptime_ms_total` which is a counter that resets every time `syncStatus` changes.